docs/defining-data-models.md

```diff
@@ -0,0 +1,126 @@
+In this section, we explain how to define `data models`, an important component of the OEF Search & Discovery. It allows agents to describe themselves and to discover the services/resources they are interested in.
```

Comment:
Minor: it should be `_data models_` (_italic_) and not ``data models``. 

The reason is the documentation was written in Sphinx, so that was supposed to be in italic.